modules/lambdas/main.tf

```diff
@@ -89,6 +89,34 @@ resource "aws_lambda_function" "delete_old_reconcile_jobs" {
   }
 }
 
+# rule to run the lambda every day
+resource "aws_cloudwatch_event_rule" "delete_old_reconcile_jobs_daily_event_rule" {
+  ## REQUIRED
+  name                = "${var.prefix}_delete_old_reconcile_jobs_daily_event_rule"
+  schedule_expression = var.orca_delete_old_reconcile_jobs_frequency_cron
+
+  ## OPTIONAL
+  description = "Triggers once per day at midnight CDT."
```

Comment:
Since this can be variable, possibly generalize the description to to something like `Scheduled execution of the ${aws_lambda_function.delete_old_reconcile_jobs.name} lambda.`

Reply:
Fixed